src/styles/stars.css

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+}
+    
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+  
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../images/star-active.svg);
+}
```

Comment:
You should make it possible to change the modifier in order to change the number of active stars. So for stars--3, stars--3 etc. it'll work as well.